string device_id
string command
---
string readings